pokedex/src/components/PokemonInfo/AddPokemonInfoModal.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import Modal from 'react-bootstrap/Modal';
+import Button from 'react-bootstrap/Button';
+import Form from 'react-bootstrap/Form';
+const { useState, useEffect } = React;
+
+export default function AddModal({ show, handleClose, pokemonInfo, handleSave }) {
+    const [nationalNumber, setNationalNumber] = useState("")
+    const [name, setName] = useState("")
+    const [photoUrl, setPhotoUrl] = useState("")
+    const [description, setDescription] = useState("")
+    const [devolvedState, setDevolvedState] = useState("")
+    const [evolvedState, setEvolvedState] = useState("")
+
+    let payload = {
+        national_num: parseInt(nationalNumber),
+        name,
+        photo_url: photoUrl, description
+    }
+    if (evolvedState.length !== 0) {
+        payload.evolved_state_pkid = parseInt(evolvedState.split(" ")[0])
```

Comment:
huh, these came out as arrays? strange

Reply:
oh it came out as "1 Pikachu" with the name at the end. Since I only needed the first one, I just splitter by space " " and then grab the first item - which is the pkid